packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -279,7 +279,8 @@ class TextMapPropagator {
         return new DatadogSpanContext({
           traceId: id(),
           spanId: null,
-          sampling: { priority }
+          sampling: { priority },
+          isRemote: true
```

Comment:
This isn't required because the default value is true but I think it's best to be explicit.